### PR TITLE
Audio files upload option

### PR DESCRIPTION
**Pull request by staffanru**

### CODE REVIEW

File: README.md
Comment (staffanru) [inline README.md:26]: Prettier changed this. Hope that is ok.

Comment (judyfong) [reply README.md:26]: Yes as long as you add a prettier github action PR next. So that the repo overall is consistent. Otherwise it doesn't make sense to have random prettier changes throughout this repo's changes

File: frontend/src/components/frontpage/index.tsx
Comment (judyfong) [inline frontend/src/components/frontpage/index.tsx]: This comment is incorrect. Processing literally means it should upload, but the button itself should no longer be possible. The code is correct though.

File: frontend/src/components/frontpage/index.tsx
Comment (judyfong) [inline frontend/src/components/frontpage/index.tsx]: This isn't a very useful console log.

File: frontend/src/components/frontpage/index.tsx
Comment (judyfong) [inline frontend/src/components/frontpage/index.tsx]:
```suggestion
            console.log(files);
```

File: frontend/src/components/frontpage/index.tsx
Comment (judyfong) [inline frontend/src/components/frontpage/index.tsx]:
```suggestion
```

File: frontend/src/components/thanks/index.tsx
Comment (judyfong) [inline frontend/src/components/thanks/index.tsx]:
```suggestion
                    <h2>Takk fyrir að taka þátt!</h2>
```
Not your spelling mistake but it'd be embarrassing if the podcasters notice

**Review by judyfong — CHANGES_REQUESTED** [review]
The code itself works great. I approve.

There are a few small changes that should be done to the code, cleaning up and whatnot. Those are addressed line by line. I suggest that you make those changes. Then merge this branch to master when you're done, and update production of course.

File: frontend/src/components/frontpage/index.tsx
Comment (judyfong) [reply frontend/src/components/frontpage/index.tsx]: These variables don't seem to be used anymore.

File: README.md
Comment (staffanru) [reply README.md:26]: Cool, I really like that so I'll add it next

File: frontend/src/components/frontpage/index.tsx
Comment (staffanru) [reply frontend/src/components/frontpage/index.tsx]: True! Removing...

File: frontend/src/components/thanks/index.tsx
Comment (staffanru) [reply frontend/src/components/thanks/index.tsx]: Hahaha, yeah!

**Comment by judyfong** [comment]
If you finish these changes this week then i’d prefer to review the changes before merging. I think a second set of eyes is usually helpful.

**Comment by staffanru** [comment]
Addressed all the comments. It is still this week but very late in it. If you have time, a second pair of eyes would be awesome.

**Review by judyfong — APPROVED** [review]
Looks fine and it runs

**Pull request merged**
